openslides_backend/main.py

```diff
@@ -119,13 +120,22 @@ def sigterm_handler(signalnum: int, current_stack_frame: Any) -> None:
 
 
 def main() -> None:  # pragma: no cover
-    component = os.environ.get("OPENSLIDES_BACKEND_COMPONENT", "all")
+    env = Environment(os.environ)
+    component = env.OPENSLIDES_BACKEND_COMPONENT
     if component == "action":
-        start_action_server()
+        start_action_server(env)
     elif component == "presenter":
-        start_presenter_server()
+        start_presenter_server(env)
     elif component == "all":
-        start_them_all()
+        if is_truthy(env.OPENSLIDES_DEVELOPMENT):
```

Comment:
Please also move the old `is_dev_mode` method into the `Environment` for easier access

Reply:
Done.